app/views/users/show.html.erb

```diff
@@ -5,7 +5,9 @@
 
   <%= render 'skills_progress', user: @user, courses: @courses %>
 
-  <%= render 'project_submissions', projects: @projects unless @projects.empty? %>
+  <% if @user.projects.any? %>
```

Comment:
:+1:  I think it will be even more performant if we use `exists` instead of `any`